### DISCUSSION TITLE
How can I add one or more administrators on self hosted version?

### DISCUSSION THREAD

**Discussion by 0xtrou** [logto-io/logto#6188] [Q&A]
Hey guys, I want to add more admin accounts for the self hosted version, is it possible and how to do it?

**Comment by charIeszhao** [comment]
> Hey guys, I want to add more admin accounts for the self hosted version, is it possible and how to do it?

Yes, it is possible. Just add it in the DB

**Reply by 0xtrou** [reply]
Cool thanks bruh
